generators/lib/distros/octane.js

```diff
@@ -31,21 +32,29 @@ function init() {
     done();
   };
 
-  module.modifyComposer = function(yo, options, localComposer, isNewProject, done) {
+  module.loadComposer = function(yo, options) {
+    var composer = {};
+    var file = yo.templatePath('drupal/drupal/' + options.drupalDistroRelease + '/composer.json');
+    if (gadget.fsExistsSync(file)) {
+      composer = yo.fs.readJSON(file);
+      // Octane gets it's version of Drupal Core from Lightning.
+      // To minimize conflicts with the rest of the generator process, we defer
+      // removing this until the last time Octane can reasonably impose this logic.
+      delete composer.require['drupal/core'];
+    }
+    return composer;
+  }
+
+  module.modifyComposer = function(yo, options, localComposer, isNewProject, done, cb) {
```

Comment:
Document what "cb" is or give it a better name.  I have no idea what it stands for.